src/hooks/useWidth.js

```diff
@@ -0,0 +1,12 @@
+import useMediaQuery from '@material-ui/core/useMediaQuery';
```

Comment:
Some code comments on what this is doing at a glance would be nice.

Reply:
Okay will add some comments,  Material-UI did have a HOC call withWidth, that would effectively tell you whether the current page size was xs, sm, md, lg ,xl. However when they created hooks versions of their components they didn't create a useWidth hook, they do however have this snippet on their website as an example  of how you could make your own. I will add this as a comment